### PR TITLE
added centralized_webservice template and test file. Tests are not fi…

### PR DESCRIPTION
**Pull request by AGuyNextDoor**
…nished

### CODE REVIEW

**Comment by AGuyNextDoor** [comment]
https://github.com/fewlinesco/sparta_courses/pull/330

File: 03_backend_developement/01_HTTP/05_centralized_webservice/.test_utils/readcode.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        .trim();
+
+      if (studentCode.length) {
+        resolve(studentCode);
+      } else {
+        reject("File does not contain any code")
+      }
+    });
+  });
+};
```
Comment (Fenntasy) [inline 03_backend_developement/01_HTTP/05_centralized_webservice/.test_utils/readcode.js]: Did you change something from the original file?
All files that uses this code are currenlty looking for the same path (random example: `02_programming_fundamentals/03_algorithms_day_1/05_play_with_for_loop_bonus/__tests__/06_sea.test.js`) So I don't see reason to duplicate it if nothing's changed.

But rereading the code, do you still need this function? As you require the student's file and run their function directly, it should not be needed anymore.

File: 03_backend_developement/01_HTTP/05_centralized_webservice/shell_functions/fetch_comments.sh [hunk 2]
```diff
@@ -0,0 +1,2 @@
+touch shell_functions/fetch_comments.result
+curl http://jsonplaceholder.typicode.com/comments > shell_functions/fetch_comments.result
```
Comment (Fenntasy) [inline 03_backend_developement/01_HTTP/05_centralized_webservice/shell_functions/fetch_comments.sh]: Just to be sure, these are not to be written by students but to be used directly?
If yes, maybe you could explain it in the exercise explanation on `sparta_courses`?

Comment (AGuyNextDoor) [reply 03_backend_developement/01_HTTP/05_centralized_webservice/shell_functions/fetch_comments.sh]: Nope ! They are given, and they are already seen in the previous exercises

**Review by Fenntasy — APPROVED** [review]

**Pull request merged**